modules/stats/stats.py

```diff
@@ -357,6 +276,19 @@ def calculate_statistics_for_plots(self, closed_trades, open_trades):
             # Save buy/sell signals
             self.buypoints[trade.pair].append(trade.opened_at)
 
+    def generate_open_trades_results(self, open_trades: [Trade]) -> list:
```

Comment:
```suggestion
    def calculate_results_open_trades(self, open_trades: [Trade]) -> list:
```
Calculate or generate both work I guess, this order in my opinion is a bit nicer

Reply:
changing to 'get_left_open_trades_results'

Reply:
done! closing convo